### ISSUE TITLE
Gabii WebGL UX

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#1585]
- [x] If user is on a mobile or tablet, do not load the WebGL and let them know that the WebGL is not compatible.
- [x] Create loading indicators while WebGL loads

**Comment by jmcglone** [comment]
User flow for clicking the "3D Model" button in the navigation menu:

![img_0100](https://user-images.githubusercontent.com/1686111/36812900-a59ff37a-1ca0-11e8-82f3-8f9846a370d1.jpg)

![img_0101](https://user-images.githubusercontent.com/1686111/36812902-a9d90b66-1ca0-11e8-8d55-4cc851491897.jpg)

**Comment by conorom** [comment]
I have a recollection that the loading screen was going to be built into the game itself, once it was compiled with a new Unity version or something?

**Comment by jmcglone** [comment]
@conorom I spoke with the Gabii authors today and they were very opposed to the idea of not auto-loading the WebGL file upon initialization of the e-reader. I have modified the acceptance criteria based on their input.

**Comment by conorom** [comment]
As a starting point I tried to fix the existing progress stuff by adding CSS/image assets to the Rails pipeline. I then experimented with the above fancy loader, which is very out-of-date. The whole DOM thing it's doing does not apply to our Unity version. Even the targeting of the `canvas`div for the game content isn't the same any more. However, there is a thread that eventually gets to the point of people trying to convert that exact solution for 2017 Unity:
https://forum.unity.com/threads/better-loading-indicator-in-webgl.335481/

There are *no* ready/working solutions there, however, and working in Rails makes it harder again. Given the requirement of including more JS stuff and new assets, and the problems we still have to fix, I think a fancier loading bar is definitely a "later if at all" thing. Too much flux.

The custom onProgress method in the PR seems to get the built-in loader working just fine.
